examples/helm/kanister/kanister-elasticsearch/kanister/elasticsearch-blueprint.yaml

```diff
@@ -47,15 +47,15 @@ actions:
           host_name="{{ .Object.metadata.labels.release }}-kanister-elasticsearch-client.{{ .StatefulSet.Namespace }}.svc.cluster.local"
           kando location pull --profile '{{ toJson .Profile }}' --path '{{ .ArtifactsIn.cloudObject.KeyValue.backupLocation }}' - | gunzip | elasticdump --bulk=true --input=$ --output=http://${host_name}:9200
   delete:
-    type: StatefulSet
+    type: Namespace
     inputArtifactNames:
     - cloudObject
     phases:
     - func: KubeTask
       name: deleteFromObjectStore
       args:
-        namespace: "{{ .StatefulSet.Namespace }}"
-        image: "kanisterio/es-sidecar:0.29.0"
+        namespace: "{{ .Namespace.Name }}"
+        image: kanisterio/kanister-tools:0.29.0
```

Comment:
Didn't have to change the image, but figured this would be more robust if es-sidecar image ever went away.